packages/devtools_app/integration_test/test/live_connection/memory_screen_helpers.dart

```diff
@@ -0,0 +1,135 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/devtools_app.dart';
+import 'package:devtools_app/src/screens/memory/panes/control/primary_controls.dart';
+import 'package:devtools_app/src/screens/memory/panes/diff/widgets/snapshot_list.dart';
+import 'package:devtools_app/src/shared/console/widgets/console_pane.dart';
+import 'package:devtools_test/helpers.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+// / Prepares the UI of the memory screen so that the eval-related elements are
+// / visible on the screen for testing.
+Future<void> prepareMemoryUI(
+  WidgetTester tester, {
+  bool closeBannerWarning = true,
+  bool collapseChart = true,
+  bool makeConsoleWider = false,
+  bool openDiff = true,
+}) async {
+  // Open memory screen.
+  await switchToScreen(
+    tester,
+    tabIcon: ScreenMetaData.memory.icon!,
+    screenId: ScreenMetaData.memory.id,
+  );
+
+  if (closeBannerWarning) {
+    await tapAndPumpWidget(
+      tester,
+      find.descendant(
+        of: find.byType(BannerWarning),
+        matching: find.byIcon(Icons.close),
+      ),
+    );
+  }
+
+  if (collapseChart) {
+    await tapAndPumpWidget(tester, find.text(PrimaryControls.memoryChartText));
+  }
+
+  if (makeConsoleWider) {
+    // The distance is big enough to see more items in console,
+    // but not too big to make classes in snapshot hidden.
+    const dragDistance = -320.0;
+    await tester.drag(
+      find.byType(ConsolePaneHeader),
+      const Offset(0, dragDistance),
+    );
+    await tester.pumpAndSettle();
+  }
+
+  // For the sake of this test, do not show extension screens by default.
+  preferences.devToolsExtensions.showOnlyEnabledExtensions.value = true;
```

Comment:
why do we need this?

Reply:
I do not know. I just moved it as it was, from a test to helpers.

Reply:
I do not see where in the diff this was copied from. This shouldn't be necessary so let's remove it.